test/e2e/tests/solana/common-solana.ts

```diff
@@ -1594,6 +1595,7 @@ export async function withSolanaAccountSnap(
       | Promise<MockedEndpoint[] | MockedEndpoint>
       | MockedEndpoint[]
       | MockedEndpoint;
+    fixtureModifications?: (builder: FixtureBuilder) => FixtureBuilder;
```

Comment:
withFixtures is kind of an overloaded term already, but what if we used `withFixtures` or `withFixtureBuilder` here?